Modelica/Mechanics/MultiBody/Visualizers.mo

```diff
@@ -829,36 +829,7 @@ with respect to frame_a (vector from the origin of frame_a to the arrow tail).
           thickness=0.5));
       annotation (
        obsolete = "Obsolete model - use ground visualization feature in Modelica.Mechanics.MultiBody.World, or use model Modelica.Mechanics.MultiBody.Visualizers.Rectangle instead",
-       Icon(coordinateSystem(preserveAspectRatio=true, extent={{-100,
-                -100},{100,100}}), graphics={
-            Polygon(
-              points={{-20,60},{-100,-32},{20,-92},{100,8},{-20,60}},
-              fillColor={0,255,0},
-              fillPattern=FillPattern.Solid),
-            Text(
-              extent={{2,82},{44,54}},
-              textString="z"),
-            Polygon(
-              points={{100,8},{100,-4},{20,-104},{20,-92},{100,8}},
-              fillColor={0,255,0},
-              fillPattern=FillPattern.Solid),
-            Polygon(
-              points={{-100,-32},{-100,-44},{20,-104},{20,-92},{-100,-32}},
-              fillColor={0,255,0},
-              fillPattern=FillPattern.Solid),
-            Ellipse(
-              extent={{-11,-7},{-1,-12}},
-              fillColor={255,255,255},
-              fillPattern=FillPattern.Solid),
-            Line(
-              points={{-6,-10},{-6,88}}),
-            Polygon(
-              points={{-6,102},{-14,72},{2,72},{-6,102}},
-              fillPattern=FillPattern.Solid),
-            Text(
-            extent={{-150,-105},{150,-145}},
-            textString="%name",
-            textColor={0,0,255})}), Documentation(info="<html>
+       Icon(coordinateSystem(preserveAspectRatio = true, extent = {{-100, -100}, {100, 100}}, initialScale = 0.1, grid = {10, 10}), graphics = {Polygon(lineColor = {255, 255, 255}, fillColor = {126, 181, 78}, fillPattern = FillPattern.Solid, points = {{-100, -30}, {20, -90}, {100, 0}, {-10, 40}}), Text(textColor = {64, 64, 64}, extent = {{20, 70}, {60, 100}}, textString = "z", horizontalAlignment = TextAlignment.Left), Polygon(lineColor = {255, 255, 255}, fillColor = {14, 111, 1}, fillPattern = FillPattern.Solid, points = {{100, -10}, {20, -100}, {20, -90}, {100, 0}}), Polygon(lineColor = {255, 255, 255}, fillColor = {14, 111, 1}, fillPattern = FillPattern.Solid, points = {{-100, -40}, {20, -100}, {20, -90}, {-100, -30}}), Line(origin = {6, -8}, points={{-6,-10},{-6,108}}), Polygon(origin = {6, 0}, points={{-6,102},{-14,72},{2,72},{-6,102}}, fillPattern=FillPattern.Solid), Text(textColor = {64, 64, 64}, extent = {{-150, -145}, {150, -105}}, textString = "%name")}), Documentation(info = "<html>
```

Comment:
Any reason to set initialScale and grid?

Reply:
@beutlich Sorry for bother you. Could you check my question?

Reply:
This is for @maltelenz .

Reply:
No, I don't see a reason to set initialScale or grid.

Reply:
Can you please create a new PR to remove it. Thanks.

Reply:
Will do.

Reply:
See PR #3105